content/zh/common/kubernetes-versions.md

```diff
@@ -5,5 +5,5 @@ _build:
 
 | 安装工具     | KubeSphere 版本 | 支持的 Kubernetes 版本                                       |
 | ------------ | --------------- | ------------------------------------------------------------ |
-| KubeKey      | v3.1.0          | v1.17.0, v1.17.4, v1.17.5, v1.17.6, v1.17.7, v1.17.8, v1.17.9, v1.18.3, v1.18.5, v1.18.6, v1.18.8, v1.19.0, v1.19.8, v1.19.9, v1.20.4 |
-| ks-installer | v3.1.0          | v1.17.x, v1.18.x, v1.19.x, v1.20.x                           |
+| KubeKey      | v3.1.1          | v1.17.0, v1.17.4, v1.17.5, v1.17.6, v1.17.7, v1.17.8, v1.17.9, v1.18.3, v1.18.5, v1.18.6, v1.18.8, v1.19.0, v1.19.8, v1.19.9, v1.20.4 |
```

Comment:
Please add `v1.20.6` to `Supported Kubernetes versions`.
<img width="810" alt="Xnip2021-07-21_15-18-43" src="https://user-images.githubusercontent.com/75110798/126448035-c90af389-2aac-4778-b8d7-08111f0140a5.png">

Reply:
Added. Thanks!